### PR TITLE
Remove DDL files and introduce a SchemaManager component

### PR DESCRIPTION
**Pull request by robertsami**

### CODE REVIEW

File: src/com/oltpbenchmark/api/SchemaManager.java [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+        LOG.info(sql);
+        db_connection.createStatement().execute(sql);
+    }
+
+    void create() throws SQLException {
+        for (Table t : tables.values()) {
+            execute(t.getDropDdl());
+        }
+        for (Table t : tables.values()) {
+            execute(t.getCreateDdl());
```
Comment (psudheer21) [inline src/com/oltpbenchmark/api/SchemaManager.java:249]: I believe execute("create table ....."); would have been a bit simpler.
Not sure if we would ever need the columns/table classes other than for creating the tables on the database.

**Review by psudheer21 — COMMENTED** [review]
I just have one comment. Let me know what you think.

Regards
Sudheer

File: src/com/oltpbenchmark/api/SchemaManager.java (hunk 1 above)
Comment (robertsami) [reply src/com/oltpbenchmark/api/SchemaManager.java:249]: the code being replaces was using hsqldb to introspect the schema of an in memory replica of the database to create insert statements with the appropriate number of wildcards. by separating column defs out, we can still support this

**Review by psudheer21 — APPROVED** [review]
Looks Good!!!

**Pull request merged**